src/social-network-provider/twitter.com/ui/index.ts

```diff
@@ -21,12 +21,12 @@ export const instanceOfTwitterUI = defineSocialNetworkUI({
     i18nOverwrite: {
         en: {
             additional_post_box__encrypted_post_pre: {
-                message: '#Maskbook ([I:b])\nDecrypt this tweet with maskbook.com @ProjectMaskbook.\n—§— /* {$1} */',
+                message: '#Maskbook ([I:b])\nDecrypt this tweet with maskbook.com @ProjectMaskbook.\n—§— $1',
             },
         },
         zh: {
             additional_post_box__encrypted_post_pre: {
-                message: '#Maskbook ([I:b])\n使用 maskbook.com @ProjectMaskbook 解密这条推文。\n—§— /* {$1} */',
+                message: '#Maskbook ([I:b])\n使用 maskbook.com @ProjectMaskbook 解密这条推文。\n—§— $1',
```

Comment:
I think `/* */` should stay... It indicates it is a comment between them...

Reply:
> I think `/* */` should stay... It indicates it is a comment between them...

Here is the combinations, and I think you are right let's add `/* */` stuff back?

![image](https://user-images.githubusercontent.com/52657989/71442977-5de80400-2743-11ea-9690-bfc4fd2a0ab0.png)